### PR TITLE
opening and validation of existing configuration file

### PR DESCRIPTION
**Pull request by kamxgal**

### CODE REVIEW

File: model/Configuration.schema [hunk 1]
```diff
@@ -0,0 +1,71 @@
+{
+    "type" : "object",
+    "properties" : {
+        "num_trajectories" : {"type" : "integer", "minimum": 0},
+        "population_path" : {"type" : "string"},
```
Comment (tozanski) [inline model/Configuration.schema:5]: I am not sure if this is included, but perhaps the string should be a path or at least non-empty string

File: model/ProjectSettings.py [hunk 2]
```diff
@@ -3,17 +3,24 @@
 from typing import Union
 from PyQt5.QtQml import qmlRegisterType
 from PyQt5.QtCore import QObject, QMetaType, pyqtProperty, pyqtSignal, QVariant
+import logging
```
Comment (tozanski) [inline model/ProjectSettings.py:6]: `logging` seem to be unused in any of the files

File: model/TanhParameters.schema [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "type" : "object",
+    "parameters" : {
+        "scale": {"type" : "integer", "minimum": 0},
```
Comment (tozanski) [inline model/TanhParameters.schema:4]: scale is real >0

File: model/TanhParameters.schema [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+        "loc": {"type" : "integer", "minimum": 0},
```
Comment (tozanski) [inline model/TanhParameters.schema:5]: I believe it is real without any limits

File: model/TanhParameters.schema [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+        "weight_detected": {"type" : "integer", "minimum": 0},
+        "weight_deaths": {"type" : "integer", "minimum": 0},
```
Comment (tozanski) [inline model/TanhParameters.schema:7]: both weights can be real >0

File: model/TanhParameters.schema (hunk 3 above)
Comment (tozanski) [reply model/TanhParameters.schema:4]: after a second thought maybe negative values also make sense 🤔  @eMaerthin ?

File: model/Configuration.schema (hunk 1 above)
Comment (kamxgal) [reply model/Configuration.schema:5]: > I am not sure if this is included, but perhaps the string should be a path or at least non-empty string

Maybe empty string or string which ends with ".jld2"? Wouldn't empty string be useful if someone wanted to save a "template" configuration?

Comment (tozanski) [reply model/Configuration.schema:5]: OK, let's leave it as it is now

File: model/TanhParameters.schema (hunk 3 above)
Comment (kamxgal) [reply model/TanhParameters.schema:4]: Is scale greater than 0 or greater or equal than 0?

File: model/TanhParameters.schema (hunk 5 above)
Comment (kamxgal) [reply model/TanhParameters.schema:7]: again: >0 or >=0?

File: model/TanhParameters.schema (hunk 3 above)
Comment (tozanski) [reply model/TanhParameters.schema:4]: scale of `0`might be difficult in numerics, but after a second thought maybe negative values also make sense 🤔  
Maybe let's make it larger than 0 and real

File: model/TanhParameters.schema (hunk 5 above)
Comment (tozanski) [reply model/TanhParameters.schema:7]: here 0 is allowed

File: model/TanhParameters.schema (hunk 3 above)
Comment (eMaerthin) [reply model/TanhParameters.schema:4]: agree, scale is > 0 and real

File: model/TanhParameters.schema (hunk 4 above)
Comment (eMaerthin) [reply model/TanhParameters.schema:5]: agree

**Comment by kamxgal** [comment]
* I can add any type of additional parameters if they are necessary
* running simulation locally is something I wanted to propose as next feature to do :)

**Pull request merged**